pkg/spanner/migration.go

```diff
@@ -60,6 +60,8 @@ const (
 	StatementKindDDL            StatementKind = "DDL"
 	StatementKindDML            StatementKind = "DML"
 	StatementKindPartitionedDML StatementKind = "PartitionedDML"
+
+	MigrationKindFixedPointIterationDML MigrationKind = "FixedPointIterationDML"
```

Comment:
Some other ideas

- ConvergentDML
- RepeatedDML
- ProgressiveDML

Reply:
See comment above regarding deferring this to next PR :D